main.js

```diff
@@ -7,6 +7,12 @@ const createMenuTree = require('./menu');
 const loadData = require('./loadData');
 require('electron-debug');
 
+// Starts server with default options
+if (process.argv.includes('--server')) {
+  require('datavoyager-server/build/server'); // eslint-disable-line global-require
```

Comment:
I wonder if just doing this import it will start a server in another process.
Does user have to start another server separately?    (Hopefully not.)